### ISSUE TITLE
auto download without message or with `:x` or `:wq`

### ISSUE THREAD

**Issue by singledoggy** [kiyoon/jupynium.nvim#118]
**Is your feature request related to a problem? Please describe.**
When I enable auto-download, it doesn't work if I use `:x` or `:wq` to quit. It seems to be triggered only by the `:w` command. Additionally, if I use some external scripts to execute `:w` every few seconds, I receive too many messages.

**Describe the solution you'd like**
Is there any way to disable the messages or add a trigger for the `:wq` or `:x` commands?

**Comment by singledoggy** [comment]
**TL;DR : This issue is caused by my Vim configuration, not a bug with your plugin.**

Hi, thank you for your prompt reply. Based on your response, I investigated my ini.vim configuration file and found that some of my settings were causing other buffers to close automatically when I closed the main buffer. This is because certain plugins output syntax check information in new buffers, which becomes irrelevant once the main file is closed. 

Since I didn't see any messages, I mistakenly thought that the download was not successful.
